tools/package-steps.proj

```diff
@@ -0,0 +1,19 @@
+<Project >
+  <Target Name="Sign">
+    <MSBuild Projects="$(MSBuildThisFileDirectory)signing/sign.proj"
+             Targets="Restore" />
+
+    <MSBuild Projects="$(MSBuildThisFileDirectory)signing/sign.proj"
+             Targets="Sign"
+             Properties="ForceEvaluation=True" />
+  </Target>
+
+  <Target Name="PublishSymbolPackages">
+    <MSBuild Projects="$(MSBuildThisFileDirectory)publishing/publish.proj"
+             Targets="Restore" />
+
+    <MSBuild Projects="$(MSBuildThisFileDirectory)publishing/publish.proj"
+             Targets="PublishSymbols"
+             Properties="ForceEvaluation=True" />
```

Comment:
What does `ForceEvaluation=True` mean/do?

Reply:
Since you just called restore in the same MSBuild node, by passing a global property (any), it forces msbuild to re-evaluate the project and thus import the nuget.g.props/targets. cc: @ericstj can correct me if I'm wrong.